src/abstractdataframe/selection.jl

```diff
@@ -138,11 +142,19 @@ function normalize_selection(idx::AbstractIndex,
     end
     fun = last(sel)
     if length(c) > 3
-        newcol = Symbol(join(@views(_names(idx)[c[1:2]]), '_'), "_etc_", funname(fun))
+        if suffix
+            newcol = Symbol(join(@views(_names(idx)[c[1:2]]), '_'), "_etc_", funname(fun))
```

Comment:
Maybe store the common part in a string to avoid duplicating the code? Same below.

Reply:
OK. But do you agree with the `suffix` name of the kwarg, if yes I would finalize the PR.